src/main/java/org/javarosa/core/model/SelectChoice.java

```diff
@@ -29,6 +29,11 @@
      */
     public TreeElement copyNode;
 
+    /**
+     * The node that this choice represents. Not serialized.
+     */
+    private TreeElement item;
```

Comment:
I briefly considered trying to use `copyNode` for this. But like I said in another comment, I think we should remove all that stuff instead.